cluster/examples/kubernetes/minio/object-store.yaml

```diff
@@ -6,7 +6,7 @@ metadata:
 apiVersion: v1
 kind: Secret
 metadata:
-  name: access-keys
+  name: my-store-access-keys
```

Comment:
I've never really liked `my-store` as an instance name even though we've been using it other places. What if we just use `a` or `one` for the sample instance name? But it's just a sample so `my-store` works too.

Reply:
`my-store` is also used in the Ceph RGW example.
I think just leaving it at `my-store` and revisiting the examples at some later point would be okay from my side. What do you think?